test/cli/testutils.py

```diff
@@ -72,8 +75,82 @@ def __lookup_cppcheck_exe():
     return exe_path
 
 
+def __run_subprocess_tty(args, env=None, cwd=None, timeout=None):
```

Comment:
This code is not easy. But I don't know if there is a better way, so I accept it.